doc/tutorial/introduction/introduction_file_input.cpp

```diff
@@ -0,0 +1,63 @@
+#include <seqan3/std/filesystem>                // to create tmp directory
+#include <seqan3/alphabet/nucleotide/dna4.hpp>  // to create right datastructure in tmp file
+#include <seqan3/io/sequence_file/output.hpp>   // to create tmp file
+
+// Creating a tmp FastA file with sequences.
+// It's necessary, because some review processes in GitHub don't allow using
+// an alreadys existing file. So you can use the whole code from this file as
+// an example or the collection of snippets, shown in the introduction.
```

Comment:
This creates a temporary file to ensure that the snippet works correctly without any dependency on external files.